templates/helpers/related.js

```diff
@@ -0,0 +1,17 @@
+const Handlebars = require('handlebars');
+
+module.exports = (relation) => {
+  const url = relation.links ? relation.links.self : '#';
+  const name = relation.attributes.summary_title;
+  let role;
+
+  if (relation.attributes && relation.attributes.role) {
+    role = `(${relation.attributes.role})`;
+  } else {
+    role = '';
+  }
+
+  return new Handlebars.SafeString(
+    '<li><a href="' + url + '">' + name + '</a> ' + role + '</li>'
+  );
+};
```

Comment:
The HTML should be a handlebars template and the processing beforehand should be done by `json-to-html-data.js`